src/test/java/com/marizueva/laboratory/hw1/calculatort_tests/base_test/BaseTest.java

```diff
@@ -0,0 +1,38 @@
+package com.marizueva.laboratory.hw1.calculatort_tests.base_test;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.*;
+
+public class BaseTest {
+    protected Calculator calculator;
+
+//    @BeforeClass
+//    public void beforeClass() {
+//        calculator = new Calculator();
+//    }
+
+    @BeforeMethod
+    public void setUp() {
+        calculator = new Calculator();
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        calculator = null;
+    }
+
+    @BeforeGroups (groups = {"add_subtract", "divide_multiply"})
```

Comment:
Code duplications. You have common `before` and `after` hook for the all tests. `BeforeGroups` and `AfterGroups` could be replaced by 
```java
@BeforeMethod(alwaysRun = true)
public void setUp() {
    calculator = new Calculator();
}
```

Reply:
done